posthog/migrations/0168_per_project_access.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.5 on 2021-09-08 13:48
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0167_feature_flag_override"),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name="team", name="users",),
```

Comment:
IIRC we can't remove fields without pushing a full v2.0 version? But unsure about this. CC @mariusandra @timgl ?

Reply:
Perfect! Maybe worth adding that text snippet to the actual migration just for future reference?